cmd/server/cadence/cadence.go

```diff
@@ -145,3 +158,32 @@ func BuildCLI() *cli.App {
 	return app
 
 }
+
+func startTestWebhookEndpoint() {
+	http.HandleFunc("/", logIncomingRequest)
+
+	fmt.Printf("Starting server for testing...\n")
+	if err := http.ListenAndServe(":8080", nil); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func logIncomingRequest(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	switch r.Method {
```

Comment:
Let's set the status code of 200 for any POST request. 
Because Cadence-notification service will retry for any other status code than above(200 and 400).

See https://github.com/uber/cadence/issues/3798